include/gatekeeper_net.h

```diff
@@ -73,6 +74,10 @@ struct gatekeeper_if {
 	 */
 	rte_atomic16_t	rx_queue_id;
 	rte_atomic16_t	tx_queue_id;
+
+	/* Specify the IPv4 and IPv6 addresses of this interface. */
+	struct in_addr  ip4_addr;
+	struct in6_addr ip6_addr;
```

Comment:
Can we determine whether the IPv4 or IPv6 address is set by checking whether it's 0? If so, we may not need extra variable to record the IP address state.